signals.js

```diff
@@ -0,0 +1,35 @@
+const signals = {
+  SIGHUP: 1,
```

Comment:
I confirm that the documentation for node 6.10.3 does state that signals exist in `os.constants.signals`, but when I try it with node 6.2.2 I get:

```
const signals = os.constants.signals;
                            ^

TypeError: Cannot read property 'signals' of undefined
```

I'm going to keep using my `signals.js` for now.